File: model/Money.cs
using Model.Common;

namespace Model
{

    public class Money : PocketObject
    {
        private double _amount;

        public Money(int id, double amount) : base(id)
        {
            _amount = amount;
        }

        public double Amount
        {
            get { return _amount; }
            set { _amount = value; }
        }

        public override string ToString()
        {
            return $"Money{{amount={_amount}}}";
        }
    }
    
}

File: model/Money.py
from model.common.PocketObject import PocketObject


class Money(PocketObject):
    def __init__(self, id, amount):
        super().__init__(id)
        self._amount = amount

    def get_amount(self):
        return self._amount

    def set_amount(self, amount):
        self._amount = amount

    def __str__(self):
        return f"Money{{amount={self._amount}}}"
